src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -195,11 +195,10 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
                 id="wi-detail-desc" 
                 class="description"
                 [innerText]="
-                  workItem.attributes['system.description'] ? 
+                  !!workItem.attributes['system.description'] ?
```

Comment:
It forces a boolean out of any `type` of variable. So no matter what value `var x` holds or whatever type is it, `!!x` is a boolean with similar truthy/falsiness. Also, it can trace nested objects well for undefined nested chains, instead of getting flat-tyres and throwing errors (say you wanna check for `abc.pnq.xyz` but `abc.pnq` isn't defined at that instance).

It avoids some conditional corner-cases and is generally a best practice to use it, as all you want is a boolean check, not a type+value comparison. To avoid similar corner cases without `!!` you can as well use `try...catch` but that's lengthier as well as less efficient way to handle it.